tests/unit/broccoli/ember-app-test.js

```diff
@@ -511,6 +515,10 @@ describe('EmberApp', function() {
         'bar': 'bar',
       });
 
+      app = new EmberApp({
+        project,
+      });
+
```

Comment:
This change was required as `app` would be `undefined` otherwise. It accidentally worked before, as `app` initialized and leaking from another test before.